docs/envs.md

```diff
@@ -11,14 +11,30 @@ This document describes the environment variables that can be used to configure
+| OBSERVER_WALLET                | String               | "<example>"                    | The public wallet address of the wallet being used to sign report upload transactions for Observer                              |
+| CHUNKS_DATA_PATH               | String               | "data/chunks"                  | Sets the location for chunked data to be saved. If omitted, chunked data will be stored in the `data` directory                 |
+| CONTIGUOUS_DATA_PATH           | String               | "data/contiguous"              | Sets the location for contiguous data to be saved. If omitted, contiguous data will be stored in the `data` directory           |
+| HEADERS_DATA_PATH              | String               | "data/headers"                 | Sets the location for header data to be saved. If omitted, header data will be stored in the `data` directory                   |
+| SQLITE_DATA_PATH               | String               | "data/sqlite"                  | Sets the location for sqlite indexed data to be saved. If omitted, sqlite data will be stored in the `data` directory           |
+| TEMP_DATA_PATH                 | String               | "data/tmp"                     | Sets the location for temporary data to be saved. If omitted, temporary data will be stored in the `data` directory             |
+| LMDB_DATA_PATH                 | String               | "data/LMDB"                    | Sets the location for LMDB data to be saved. If omitted, LMDB data will be stored in the `data` directory                       |
+| CHAIN_CACHE_TYPE               | String               | "redis"                        | Sets the method for caching chain data, defaults redis if gateway is started with docker-compose, otherwise defaults to LMDB    |
+| REDIS_CACHE_URL                | String (URL)         | "redis://localhost:6379"       | URL of Redis database to be used for caching                                                                                    |
+| REDIS_CACHE_TTL_SECONDS        | Number               | 28800                          | TTL value for Redis cache, defaults to 8 hours (28800 seconds)                                                                  |
+| ENABLE_FS_HEADER_CACHE_CLEANUP | Boolean              | false                          | If true, periodically deletes cached header data                                                                                |
+| NODE_JS_MAX_OLD_SPACE_SIZE     | Number               | system default                 | Sets the memory limit, in Megabytes, for NodeJs. Default value depends on hardware                                              | 
+| SUBMIT_CONTRACT_INTERACTIONS   | Boolean              | true                           | If true, Observer will submit its generated reports to the ar.io Network. If false, reports will be generated but not submitted |
```

Comment:
I think the wording should be clarified on this one. It controls whether contract interactions will be submitted, but not whether reports will be saved.